curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60ffefd6479a3d084fb77cbc.md

```diff
@@ -7,9 +7,8 @@ dashedName: step-64
 
 # --description--
 
-Last, but not least, make the `input` for the terms and condition `inline`, then change the text color of the `terms and conditions` link element to `#dfdfe2`.
+Make the `input` for the terms and condition `inline` by adding the appropriate class in the HTML. 
```

Comment:
should be terms and conditions since that is what the label and input says

```suggestion
Make the `input` for the terms and conditions `inline` by adding the appropriate class in the HTML. 
```